src/charm.py

```diff
@@ -1162,14 +1205,10 @@ def _config_swift_plugin(self, swift_config):
         Args:
             swift_config: swift configuration parsed from wp_plugin_openstack-objectstorage_config
                 config. Use :meth:`WordpressCharm._swift_config` to get the parsed swift config.
-
-        Raises:
```

Comment:
Is it no longer possible for this exception ot occur? I couldn't spot it being removed in a change below

Reply:
I think this is removed when I merged the `sidecar` branch into this PR. I will add it back.

Reply:
Added. thanks!